master/internal/api_notebook.go

```diff
@@ -130,10 +130,13 @@ func (a *apiServer) IdleNotebook(
 func (a *apiServer) KillNotebook(
 	ctx context.Context, req *apiv1.KillNotebookRequest,
 ) (resp *apiv1.KillNotebookResponse, err error) {
+	// CHECK: do we want to depend on "can view notebook" permission to kill it?
```

Comment:
I believe we do not want to depend on view

Reply:
I brought it up in this weeks rbac sync and we do want to do that. I'll post in the rbac channel I know we had some related discussions a while back.

Reply:
https://determined-ai.slack.com/archives/C03LE5Z8R46/p1671743489958169?thread_ts=1669678264.403459&cid=C03LE5Z8R46